components/scream/ekat/tests/algorithm/lin_interp_test.cpp

```diff
@@ -176,11 +176,16 @@ TEST_CASE("lin_interp", "soak") {
 
     // Compare results
     {
+#ifdef EKAT_STRICT_FP
```

Comment:
Is this macro defined only for this test, or as part of the EKAT configuration? If the latter, maybe we need some sort of value for Strict exposed that prevents folks from having to do this #ifdef themselves. Sorry if this is a silly question--there's a lot of code in this PR. :-)

Reply:
We can define a constexpr somewhere, which evaluates to true or false depending on the ifdef. The best place would be ekat_config.h, but that would make parsing it for fortran complicated (we would need two separate ekat_config.h.in and ekat_config.f.in files). Another possible place is `scream_test_utils.hpp`, since FP stuff is mostly needed for testing.

Reply:
If you think it's worth it, that sounds good.

Reply:
You mean add it inside test utils?

Reply:
Yeah, if it's only useful to tests. But as you point out, there are complexities resulting from our multiple languages. Honestly, I could go either way--I was just trying to understand how the macro is used.

Reply:
In Fortran, one can still include `ekat_config.f` and do an ifdef. Maybe create a `logical, parameter :: fp_strict` based on that, in a helper module. But Ok, I think putting it in scream_test_utils.hpp for now is enough.